apps/pollencount/pollen_count.star

```diff
@@ -0,0 +1,318 @@
+"""
+Applet: Pollen Count
+Summary: Pollen count for your area
+Description: Displays a pollen count for your area. Enter your location for updates every 12 hours on the current conditions in your town, as well as which types of pollen are in the air today.
+Author: Nicole Brooks
+"""
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+
+DEFAULT_LOC = {
+    "lat": 40.63,
+    "lng": -74.02,
+    "locality": "",
+}
+
+COLORS = {
+    "yellow": "#D19C21",
+    "red": "#B31F0E",
+    "green": "#338722",
+}
+
+API_URL_BASE = "https://api.tomorrow.io/v4/timelines?&fields=treeIndex,weedIndex,grassIndex&apikey=d4N7IbI70OfDtNh56VihCnwi3Crg2th6&timesteps=1d&location="
```

Comment:
Is this a sensitive API key? If so, please generate a new API key and add this as a secret 🙀  https://github.com/tidbyt/pixlet/blob/main/docs/modules.md#pixlet-module-secret

Reply:
It's a free API that anyone can sign up for and has no potential I can see for misuse (the worst people could do with it is make other requests instead of making their own free account and use up the rate limit lol), but for absolute safety's sake I can do that.

Reply:
I've updated with a new, encrypted API key. Let me know :)